### DISCUSSION TITLE
ecatalogue - Catalogue Module

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#905] [Structural Changes]
This is a discussion to collate and discuss changes that we would like to see in the catalogue in EMu. These may be structural or data.

[OPEN STRUCTURAL CHANGE ISSUES](https://github.com/fieldmuseum/EMu-Documentation/issues?q=is%3Aissue+is%3Aopen+label%3A%22structural+changes%22+label%3Aecatalogue)

**Comment by rondlg** [comment]
From Zendesk ticket [#63569](https://fieldmuseum.zendesk.com/agent/tickets/63569)

Hello,

When creating a new record in the Fossil Invertebrate catalog and PE is entered into Series (GeoSeries) a number is automatically generated and appears in the Catalog # (GeoCatalogNo). Can this be turned off so no number is generated?

Paul Mayer

**Comment by rondlg** [comment]
- https://github.com/fieldmuseum/EMu-Documentation/discussions/964

**Comment by rondlg** [comment]
- https://github.com/fieldmuseum/EMu-Documentation/discussions/962

**Comment by magpiedin** [comment]
- https://github.com/fieldmuseum/EMu-Documentation/discussions/1005

**Comment by fmjjones** [comment]
Anthropology would like CatObjectCount field deprecated.

**Comment by fmjjones** [comment]
From the Ecatalogue working group March 5th: https://github.com/fieldmuseum/EMu-Documentation/discussions/1017
Decision: Anthro is using the Original Count field for Items Cat and the Count field for Item inv, the fragments field is not used. Zoology, Geology would like to use the current Fragments field [PreNoOfFragments_tab] to count parts/pieces and would like to relabel it “pieces” rather than fragments.

**Comment by rbaquiran** [comment]
Question from March 26: 
Can we lockdown the Life Stage (PheStage_tab) or Sex (PheSex_tab) lookup lists by catalog? [RB-INS]
Description: Our life stages and sexes are tightly controlled in Insects & Arachnids, but I see some new values being used in other catalogs are throwing our data entry people for a loop. (F in 2377256, 2380728, 2380729, & “?” in 197620, 197644) Is it possible to limit our options in these two fields by catalog? Thanks for considering it.

**Reply by fmjjones** [reply]
The general consensus from the EMu users group meeting is supported by all.

**Comment by rondlg** [comment]
Add identifier fields to the Stratigraphy module similar to on the identification tab plus a free text field.

**Comment by cmaves3** [comment]
In Catalogue module, Specimen tab: text for the fields Collection Notes (ColCollectionNotes), Morphology (PalMorphology), and Lithology (PalLithology) should wrap to additional lines instead of being cut off. This is currently how RelNotes in Associations tab works.

**Comment by cmaves3** [comment]
On behalf of the Geology Collection Managers, I am requesting that the field PriInventory be added to the Specimen tab of the Catalogue module for the following EMu user groups: FossilVertMgr, PaleobotanyMgr, PhysicalGeologyMgr, FossilinvertsMgr

**Comment by rondlg** [comment]
Stratigraphy qualifiers same as for taxonomic identifications:  github.com/fieldmuseum/EMu-Documentation/discussions/1125

**Reply by cmaves3** [reply]
Is this dev inclusive of Qualifier Rank? @rondlg

**Comment by rondlg** [comment]
Provenience Tab - Accomodating for non-collection site attachment. 
https://github.com/fieldmuseum/EMu-Documentation/issues/5

**Reply by fmjjones** [reply]
This is addressed in version 9.1 with the introduction of the new shared Collection Tab and the new Site field ColAntSiteRef on the Provenience Tab.

**Comment by rondlg** [comment]
Deaccession Tool - https://github.com/fieldmuseum/EMu-Documentation/issues/1118

- Also referenced in efmnhtransactions

**Comment by rondlg** [comment]
`IdeTaxonRef_tab` (Taxon) 
Currently = the value in the 1st row of the idenfication table and includes the deprecated `PriSex` field
NEW = Filed As Taxa = `IdeFiledAsTax`; Type Status = `IdeTypeStatus_tab`; Sex = `PheSex_tab`

- @fmjjones which PheSex_tab row should we use? I don't see a preferred, do we need to add that or should it be a concatenation of all of them?

**Reply by rondlg** [reply]
From @fmjjones 
Janeen Jones
  11:26
A concatenation of all of them.

**Comment by rondlg** [comment]
- https://github.com/fieldmuseum/EMu-Documentation/issues/519

**Comment by rondlg** [comment]
- https://github.com/fieldmuseum/EMu-Documentation/discussions/1191

**Comment by cmaves3** [comment]
Geology has auto-incrementing Catalog #s partially implemented in the various Catalogs. They would like it to be implemented across all Catalogs. The PF series should be auto-incrementing at around PF 18000 but instead is around PF 60605 - that should be reset.

**Comment by rbaquiran** [comment]
Just to make sure these auto-incremented catalog numbers are just being
discussed for Geology, right?

**Reply by fmjjones** [reply]
Yes, these are just being discussed across the Geology catalogues.

**Comment by cmaves3** [comment]
On the Locations tab, Movement History table: the character limit for the Location field should be increased. Vert Paleo Locations summaries can be up to 99 characters long

**Reply by cmaves3** [reply]
The issue this was attempting to address is that we are not seeing the full location string. The issue seems to actually be that the value is pulling from **MovLocationSummary** instead of **MovLocationSummaryLocal_tab** (as found in Internal Movements module > Audit tab). The line in the Internal Movements record does the same thing. For Fossil Verts, this means that Cabinet and Drawer are not visible in the string.

**Comment by chungkamei** [comment]
EMu group HerpsMgr: I'd like to be able to auto generate catalog numbers, as I am able to for Transactions (loans)) by pressing "+" and Tab. Is that something you can help me with? Mammals collections are able to do that, and I think that's a very helpful tool to be able to do that.

**Comment by magpiedin** [comment]
Identifications tab -- move/add the `IdeFiledAsRef` etaxonomy attachment (single-value) to the standard Identifications query + display tab (from AllSpeRRCTab)

**Comment by cmaves3** [comment]
FossilVertsMgr has the tabs "Palaeo SQS" (QryPalSQSTab) and "DwC 1.2 (1)" (QryDar1Tab) where you can search by several Taxonomy terms: Kingdom, Phylum, Class, Order, Family, Genus, Species, Subspecies. ("Palaeo SQS" is not present in Display view)
We would like to be able to search by and view additional taxonomy terms: Subphylum, Superclass, Subclass, Superorder, Suborder, Infraorder, and Subfamily
Ideally, all ranks of the taxon hierarchy from Phylum down to Subspecies will be searchable and viewable on a single tab. It should also account for instances where there are multiples of a rank (i.e. Diapsida, irn 554097, is a Class under Reptillia, also a Class).

**Comment by cmaves3** [comment]
Label for PriAccessionNumber on tab QrySpeGTab is currently "Received From" indicating a Parties attachment. It is actually the attachment for the Accession record in Transactions so should be labeled as such.

**Comment by zkachian** [comment]
I would like to define the summary string logic for all Action user groups when viewing Catalog records where Catalog (CatCatalog) = "Amphibians and Reptiles" AND Catalog Subset (CatCatalogSubset) = "Tissue" OR "Sighting". Currently, the summary string logic is not defined for these records so they just appear blank and only the specimen records display any data (screenshot below).

For Catalog Subset = Tissue, I would like to use the same logic as is currently being used for herps specimen catalog records. 
For Catalog Subset = Sighting, I would like to use the same logic as is currently being used for herps specimen catalog records, except that I would like to remove "FMNH #" from the summary string, since sighting records will never have a FMNH #.

Please let me know if I can provide any additional info that would be helpful.

<img width="792" height="559" alt="image" src="https://github.com/user-attachments/assets/7ca9a07c-e6ec-4f02-86fe-04589c5daa13" />

**Comment by J-a-m-i-e** [comment]
Adding here as a request from testing v.9 for Preps and Custody tabs: We need fields in the Parent record on the Preps tab (and carried through on the newer Custody (AllCatSharedCusTab) tab for the PreTotalCount, PreTotalOriginalCount, and CatItemsSubPrep fields) that calculates the sum of values for child records for the Items Sub (CatItemsSubPrep) field, the Original Count (PreOriginalCount_tab) field, and the Count (PreCount_tab) field. This is something we had previously in v.5 with the CatItemsCat, CatItemsInv, and CatItemsSub fields before they were deprecated when the Preps tab was introduced. Happy to discuss further how this could look and work better for us.

**Comment by J-a-m-i-e** [comment]
Another request: the AnthroSQS (QryAntATab) tab we in Anthro use in the Catalog module has the old Site Link (ColSiteLocation) field, so you have to search from the new Site Link (ColAntSite) field from the Provenience (QryProTab) tab. We would like the old Site Link field on the AnthroSQS (QryAntATab) tab to be updated to the new one (ColAntSite).

**Comment by fmjjones** [comment]
During[ Version 9 testing](https://docs.google.com/spreadsheets/d/1S06Dk6Pcg216T8PxGkQFxaPiKBouzyt49xNXa7j9FG4/edit?gid=0#gid=0) Fossil Verts has requested:
Can we add Accepted By [LocAcceptedBy] to the movement history table?
